packages/scaffolding-cli/src/__tests__/domain/config/worker_maps/ssr_aks_tfs.test.ts

```diff
@@ -15,14 +15,14 @@ let biz: BusinessSection = <BusinessSection>{
 
 let cloud: CloudSection = <CloudSection>{
     region: "uksouth",
-    resource_group: "my-rg"
+    resourceGroup: "my-rg"
 }
 
 let files: Array<BuildReplaceInput> = [
     {
         files: ["**/package.json"],
         values: {
-            "project_name": proj_name
+            "PROJECT_NAME": projName
```

Comment:
`project_name` can be lower case - we should adjust the package.json in SSR instead as it's node/npm package.json convention to have no uppercase characters in project name value

Reply:
AB#1978